build/helper.py

```diff
@@ -231,14 +240,20 @@ def get_library_call_parameter_types_snippet(parameters_list):
 def _get_output_param_return_snippet(output_parameter, parameters):
     '''Returns the snippet for returning a single output parameter from a Session method, i.e. "reading_ctype.value"'''
     assert output_parameter['direction'] == 'out', pp.pformat(output_parameter)
+    return_type_snippet = ''
+    if output_parameter['enum'] is not None:
+        return_type_snippet = 'enums.' + output_parameter['enum'] + '('
```

Comment:
this is fine... but if we find ourselves appending enums. and python_types. all over the place, then perhaps we should store the appended string in the metadata.